DigitalLearningSolutions.Web/Controllers/TrackingSystem/Delegates/DelegateProgressController.cs

```diff
@@ -50,22 +50,21 @@ ISearchSortFilterPaginateService searchSortFilterPaginateService
             this.searchSortFilterPaginateService = searchSortFilterPaginateService;
         }
 
-        public IActionResult Index(int progressId, DelegateAccessRoute accessedVia, int? returnPage)
+        public IActionResult Index(int progressId, DelegateAccessRoute accessedVia)

```

Comment:
Ah there was a conversation with just myself and Steve about consistency, and Kevin agreed to remove the back link from this page.
Since the page links to a number of sub-pages, it causes us to either have to preserve the return page query for all those sub pages too. If we don't do this the back link behaviour becomes inconsistent.